src/wp-includes/class-wp-scripts.php

```diff
@@ -771,6 +788,23 @@ public function add_data( $handle, $key, $value ) {
 		return parent::add_data( $handle, $key, $value );
 	}
 
+	/**
+	 * Check all handles for any delayed inline scripts.
+	 *
+	 * @return bool True if script present. False if empty.
```

Comment:
We are checking inline script so this need to be changes.

```suggestion
	 * @return bool True if the inline script present, otherwise false.
```